src/HotkeysProvider.tsx

```diff
@@ -30,44 +30,31 @@ interface Props {
 }
 
 export const HotkeysProvider = ({ initiallyActiveScopes = ['*'], children }: Props) => {
-  const [internalActiveScopes, setInternalActiveScopes] = useState(
-    initiallyActiveScopes?.length > 0 ? initiallyActiveScopes : ['*']
-  )
+  const [internalActiveScopes, setInternalActiveScopes] = useState(initiallyActiveScopes)
```

Comment:
In my opinion it would be more user friendly to have all scopes activated by default. At least this is what I would expect as a user. What do you think?

Reply:
Well, in my opinion, if the user explicitly gives an empty array, I think they would be expecting no scopes to be active at all. On the other hand, if they pass nothing at all (undefined), they would indeed have all scopes active.

I think this is more fitting, as you may want to initialize with no scopes active at all, and passing an empty array would accomplish this behavior.

Reply:
Ah you are right.